docs/src/components/ExampleCode.css

```diff
@@ -0,0 +1,3 @@
+.LiveEditor__textarea__notExpanded textarea {
+  cursor: zoom-in !important;
```

Comment:
The library is also trying to set cursor, so need to override that.

The library doesn't provide a way to add styles for the underlying textarea. This is potentially brittle, but I doubt future versions will move away from using a textarea, so this feels relatively safe.